lightning_hpo/commands/sweep/list_sweeps.py

```diff
@@ -0,0 +1,53 @@
+from lightning_hpo.components.servers.db.models import GeneralModel
+from typing import List
+from lightning.app.utilities.commands import ClientCommand
+import requests
+import abc
+from rich.console import Console
+
+from rich.table import Table
+
+
+class Formatable(abc.ABC):
+    @abc.abstractmethod
+    def as_table(self) -> Table:
+        pass
+
+
+class SweepsList(Formatable):
+    def __init__(self, sweeps: List[dict]):
+        self.sweeps = sweeps
+
+    def as_table(self) -> Table:
+        # TODO: Improve the table format.
+        table = Table(
```

Comment:
Let's use a Tree format with the Trial information.

Reply:
There's a lot of information, what information do you want to display ? e.g. this is the json of one sweep with 3 trials 

```json

{
   "requirements":[
      
   ],
   "script_args":[
      
   ],
   "trials":{
      "0":{
         "best_model_score":0.07500000298023224,
         "monitor":"val_acc",
         "best_model_path":"None",
         "status":"succeeded",
         "params":{
            "params":{
               "model.lr":0.08025794913236321
            }
         },
         "exception":"None"
      },
      "1":{
         "best_model_score":0.09375,
         "monitor":"val_acc",
         "best_model_path":"None",
         "status":"succeeded",
         "params":{
            "params":{
               "model.lr":0.099850192637455
            }
         },
         "exception":"None"
      },
      "2":{
         "best_model_score":0.125,
         "monitor":"val_acc",
         "best_model_path":"None",
         "status":"succeeded",
         "params":{
            "params":{
               "model.lr":0.033720632727005546
            }
         },
         "exception":"None"
      }
   },
   "sweep_id":"mansy-942286bf",
   "n_trials":3,
   "trials_done":3,
   "framework":"pytorch_lightning",
   "num_nodes":1,
   "direction":"maximize",
   "status":"succeeded",
   "id":1,
   "distributions":{
      "model.lr":{
         "distribution":"uniform",
         "params":{
            "params":{
               "low":0.001,
               "high":0.1
            }
         }
      }
   },
   "script_path":"train.py",
   "simultaneous_trials":1,
   "url":"None",
   "cloud_compute":"cpu",
   "logger":"wandb"
}
```